src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -10502,5 +10499,105 @@ .maxstack  1
     }
 ");
         }
+
+        [Fact]
+        public void Records_AddWellKnownMember()
+        {
+            var source0 =
+@"
+namespace N
+{
+    record R(int X)
+    {
+    }
+}
+";
+            var source1 =
+@"
+namespace N
+{
+    record R(int X)
+    {
+        protected virtual bool PrintMembers(System.Text.StringBuilder builder)
+        {
+            return true;
+        }
+    }
+}
+";
+
+            var compilation0 = CreateCompilation(new[] { source0, IsExternalInitTypeDefinition }, options: ComSafeDebugDll);
+            var compilation1 = compilation0.WithSource(new[] { source1, IsExternalInitTypeDefinition });
+
+            var printMembers0 = compilation0.GetMember<MethodSymbol>("N.R.PrintMembers");
+            var printMembers1 = compilation1.GetMember<MethodSymbol>("N.R.PrintMembers");
+
+            var v0 = CompileAndVerify(compilation0, verify: Verification.Skipped);
```

Comment:
Is skipping verification the correct thing to do if verification fails on desktop? Or is further investigation needed?

Found an issue that seemed to imply this was appropriate.

Reply:
I think so. The compiler likely emits code that the legacy desktop verifier fails on, but it's actually ok.

---
In reply to: [597320232](https://github.com/dotnet/roslyn/pull/51926#discussion_r597320232) [](ancestors = 597320232)